README.md

```diff
@@ -22,42 +22,61 @@ If you have both installed, docker will be used by default.  If you
 want to use a specific container runtime, set `XCPNG_OCI_RUNNER` to
 the docker-compatible command to use (typically `podman` or `docker`).
 
+You'll need to install git-lfs to be able to download the source tarballs from
+git, otherwise when running xcp-ng-dev, it won't be able to extract the sources.
+
+## Installation
+
+Clone this repository and install the `xcp-ng-dev` package:
+
+```bash
+git clone xcp-ng/xcp-ng-dev-build-env
+cd xcp-ng-dev-build-env
+uv tool install --from . xcp-ng-dev
+```
+
+If `uv` is not available you can use other tools to install python packages,
+like `pipx install .`
+
+After this, two new commands will be available: `xcp-ng-dev-build` and
+`xcp-ng-dev`,
+
 ## Building the container image(s)
 
 You need one container image per target version of XCP-ng.
 
-Clone this repository (outside any container), then use `build.sh` to
+Clone this repository (outside any container), then use `xcp-ng-dev-build` to
 generate the images for the wanted releases of XCP-ng.
 Note that Docker and Podman store container images separately.
 
 ```
-Usage: ./build.sh {version_of_XCP_ng}
+Usage: xcp-ng-dev-build {version_of_XCP_ng}
 ... where {version_of_XCP_ng} is a 'x.y' version such as 8.0.
 ```
 
 ## Using the container
 
-Use the `run.py` script. It accepts a variety of parameters allowing for different uses:
+Use `xcp-ng-dev`. It accepts a variety of parameters allowing for different uses:
 * rebuild an existing source RPM (with automated installation of the build dependencies)
 * build a package from an already extracted source RPM (sources and spec file), or from a directory that follows the rpmbuild convention (a `SOURCES/` directory and a `SPECS/` directory). Most useful for building packages from XCP-ng's git repositories of RPM sources: https://github.com/xcp-ng-rpms.
 * or simply start a shell in the build environment, with the appropriate CentOS, EPEL and XCP-ng yum repositories enabled.
 
 ```sh
-usage: run.py [-h] [-b BRANCH] [-l BUILD_LOCAL] [--define DEFINE]
-              [-r REBUILD_SRPM] [-o OUTPUT_DIR] [-n] [-p PACKAGE] [-s SRPM]
-              [-d DIR] [-e ENV] [-v VOLUME] [--rm] [--syslog] [--name NAME]
-              [-a ENABLEREPO] [--fail-on-error]
-              ...
+usage: xcp-ng-dev [-h] [-b BRANCH] [-l BUILD_LOCAL] [--define DEFINE]
+                  [-r REBUILD_SRPM] [-o OUTPUT_DIR] [-n] [-p PACKAGE]
+                  [-s SRPM] [-d DIR] [-e ENV] [-v VOLUME] [--rm] [--syslog]
+                  [--name NAME] [--ulimit ULIMIT] [-a ENABLEREPO]
+                  [--disablerepo DISABLEREPO] [--fail-on-error]
+                  ...
```

Comment:
Do we really want to keep a `--help` copy here?  What about fully dropping it instead?

Reply:
Ok for me

Reply:
Now it's dropped